azure-container-apps/README.md

```diff
@@ -0,0 +1,371 @@
+
+## Appendix: Resource recommendations
+
+The pod for 1Password SCIM Bridge should be vertically scaled you provision a large number of users or groups. These are our default resource specifications and recommended configurations for provisioning at scale:
+
+| Volume    | Number of users | CPU   | memory |
+| --------- | --------------- | ----- | ------ |
+| Default   | <1,000          | 0.25  | 0.5Gi  |
+| High      | 1,000–5,000     | 0.5   | 1.0Gi  |
+| Very high | >5,000          | 1.0   | 1.0Gi  |
+
+If you're provisioning more than 1,000 users, update the resources assigned to the SCIM bridge container to follow these recommendations. The resources specified for the Redis container do not need to be adjusted.
+
+### Default
+
+To revert to the default specification that is suitable for provisioning up to 1,000 users, run the following command:
+
+```sh
+az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge \
+  --cpu 0.25 --memory 0.5Gi
+```
+
+### High volume deployment
+
+If you're provisioning up to 5,000 users:
+
+```sh
+az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge \
+  --cpu 0.5 --memory 1.0Gi
+```
+
+### Very high volume
+
+If you're provisioning more than 5,000 users:
+
+```sh
+az containerapp update -n $ConAppName -g $ResourceGroup --container-name op-scim-bridge \
+  --cpu 1.0 --memory 1.0Gi
+```
+
+## Get help
+
+### Region support
+
+When you create or deploy the Container App Environment, Azure may present an error that the region is not supported. You can review Azure documentation to make sure the region you selected supports [Azure Container Apps](https://azure.microsoft.com/en-ca/explore/global-infrastructure/products-by-region/?regions=all&products=container-apps).
+
+### Container App Name requirements
+
+Your Container App Name, (the `ConAppName` variable) can contain lowercase letters, numbers, and hyphens. It must be between 2 and 32 characters long, cannot start or end with a hyphen, and cannot start with a number. [Learn more about the naming rules and restrictions for Azure resources](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules#microsoftapp).
+
+### Viewing logs in Azure Container Apps
+
+Logs for a container app can be viewed from the **Log Stream**, there is a separate log stream for both containers of the revision that is active. Often reviewing the logs of the **op-scim-bridge** container can help understand any startup issues.
+
+### How to replace your `scimsession` secret in the deployment
+
+After you download a new `scimsession` file, follow the steps below to replace the existing one in your deployment.
```

Comment:
Hmm I'm not sure we need to given the context here implies they already did that. @black-bryan what do you think?

Reply:
100% it is implied that they have already download the new scimsession file. We could however link to: https://support.1password.com/scim-troubleshooting/#if-you-lose-your-bearer-token-or-session-file
But the wording on this AKB guide states redeploy your SCIM bridge, which is not always needed. 

Perhaps a separate issue is needed to alter the wording on our AKB guide to "Then [redeploy the SCIM bridge](https://support.1password.com/scim/#step-1-set-up-and-deploy-1password-scim-bridge), or replace your scimsession secret on your existing deployment."

Reply:
I'd vote to leave this as-is since it reads pretty clear to me, but @HeatherRobb let me know if you want to add a link or more info with what Bryan said in mind.

Reply:
I'd be okay to leave this as-is, if everyone thinks it's clear. I reviewed this over a longer period of time, so I expect it had just been a while since I had read the information about creating the `scimsession` file. 😄